background/test/app.test.js

```diff
@@ -43,38 +43,48 @@ describe('behavior of cloud function', function() {
     } catch (err) {
       console.log("Wasn't able to delete Google Cloud Functions");
     }
-    try {
-      cp.execSync(
-        `firebase firestore:delete --project ${projectId} -r translations`
-      );
-    } catch (err) {
-      console.log("Wasn't able to delete firestore project");
-    }
+    console.log('before deletion');
+    const db = new Firestore({
+      project: projectId,
+    });
+    await db
```

Comment:
Chaining an `await` and a `then` is generally not preferred.  You usually pick a style (async/await, or then/catch) and then use it.  A more consistent way to say this would be:
```js
const res = await db.collection('/translations').get();
for (const element of res) {
  await element.ref.delete();
}
```